content/id/docs/tasks/administer-cluster/manage-resources/memory-constraint-namespace.md

```diff
@@ -0,0 +1,272 @@
+---
+title: Mengatur Batas Minimum dan Maksimum Memori pada sebuah Namespace
+content_type: task
+weight: 30
+---
+
+
+<!-- overview -->
+
+Laman ini menunjukkan cara untuk mengatur nilai minimum dan maksimum memori yang digunakan oleh Kontainer
+yang berjalan pada sebuah Namespace. Kamu dapan menentukan nilai minimum dan maksimum memori pada objek
+[LimitRange](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#limitrange-v1-core). Jika sebuah Pod tidak memenuhi batasan yang ditentukan oleh LimitRange,
+Pod tersebut tidak dapat dibuat pada Namespace tersebut.
+
+
+
+
+## {{% heading "prerequisites" %}}
+
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+Tiap Node dalam klastermu harus memiliki setidaknya 1 GiB memori.
+
+
+
+
+<!-- steps -->
+
+## Membuat sebuah Namespace
+
+Buat sebuah Namespace sehingga sumber daya yang kamu buat pada latihan ini
+terisolasi dari komponen lain pada klatermu.
+
+```shell
+kubectl create namespace constraints-mem-example
+```
+
+## Membuat LimitRange dan Pod
+
+Berikut berkas konfigurasi untuk sebuah LimitRange:
+
+{{< codenew file="admin/resource/memory-constraints.yaml" >}}
+
+Membuat LimitRange:
+
+```shell
+kubectl apply -f https://k8s.io/examples/admin/resource/memory-constraints.yaml --namespace=constraints-mem-example
+```
+
+Melihat informasi mendetail mengenai LimitRange:
+
+```shell
+kubectl get limitrange mem-min-max-demo-lr --namespace=constraints-mem-example --output=yaml
+```
+
+Keluaran yang dihasilkan menunjukkan batasan minimum dan maksimum dari memori seperti yang diharapkan. Tetapi
+perhatikan hal berikut, meskipun kamu tidak menentukan nilai semula pada berkas konfirugasi untuk 
+LimitRange, nilai tersebut akan dibuat secara otomaris.
+
+```
+  limits:
+  - default:
+      memory: 1Gi
+    defaultRequest:
+      memory: 1Gi
+    max:
+      memory: 1Gi
+    min:
+      memory: 500Mi
+    type: Container
+```
+
+Mulai sekarang setiap Kontainer yang dibuat pada Namespace constraints-mem-example, Kubernetes
+akan menjalankan langkah-langkah berikut:
+
+* Jika Kontainer tersebut tidak menentukan permintaan dan limit memori, pasangkan nilai permintaan 
```

Comment:
Keep the API name "Container"
```suggestion
* Jika Container tersebut tidak menentukan permintaan dan limit memori, maka diberikan nilai permintaan 
```

Reply:
All resolved in e323b8c5920d6ae02c050306495e784b8e19ad9f